compiler/z/codegen/S390Instruction.cpp

```diff
@@ -2010,7 +2010,12 @@ TR::S390RILInstruction::adjustCallOffsetWithTrampoline(int32_t offset, uint8_t *
 
    // Check to make sure that we can reach our target!  Otherwise, we need to look up appropriate
    // trampoline and branch through the trampoline.
+   
+#if defined(TR_TARGET_64BIT) && defined(STRESS_TRAMPOLINES)
+   if (1)
```

Comment:
From what I remember, STRESS_TRAMPOLINES is broken beyond recognition due to total incompatibility with code cache switching. It would probably be to remove it then to staple on another ifdef as though it mattered.